pkg/system/reconciler.go

```diff
@@ -160,6 +162,7 @@ func NewReconciler(
 	r.ServiceAccount.Namespace = r.Request.Namespace
 	r.CoreApp.Namespace = r.Request.Namespace
 	r.NooBaaMongoDB.Namespace = r.Request.Namespace
+	r.PostgresDBConf.Namespace = r.Request.Namespace
```

Comment:
add it also to checkAll function in this file

Reply:
but should it be conditional there in checkAll depending on the dbtype?

Reply:
yes, there is existing condition there for dbtype

Reply:
@romayalon, thank you! Sounds good.

See https://github.com/baum/noobaa-operator/blob/39c5a4ea3cad97d10aef872aa3c86ca68cbc6966/pkg/system/reconciler.go#L270

Reply:
oh I see it's missing also in the // Set Names section,
and maybe finding a better name other than huge-pages-config is preferred since it might be used for other configurations in the future

Reply:
@romayalon,  👍 tweaked YAML level definitions, to get rid of huge-pages references on name level, looks tidier now, and allows future extensions. 

 https://github.com/noobaa/noobaa-operator/compare/3ec69ef..5d1098a

however set names change is probably not required, since the configMap name is static/constant and also referenced by the stateful state, as a volume.

WDYT?

Reply:
@baum That was what I was thinking also, I saw that there is no .name but figured out it is not needed. 
@romayalon Are we missing something?

Reply:
@baum @liranmauda it brings up the question of why even creating this Yaml file and not use an empty one that already exists and populate it before creation? is there any special reason for it?

Reply:
@romayalon @liranmauda

The configMap name is defined here:

https://github.com/baum/noobaa-operator/blob/42d9cbd07179a6afbb54e0c6d2714fc4f2532b8d/deploy/internal/configmap-postgres-db.yaml#L4

Referenced by the stateful set here:

https://github.com/baum/noobaa-operator/blob/42d9cbd07179a6afbb54e0c6d2714fc4f2532b8d/deploy/internal/statefulset-postgres-db.yaml#L70

No golang logic is required, all on YAML level.

Thank you for your review!